### PR TITLE
fix(navbar): remove dependency to ng-ovh-chatbot

### PR DESCRIPTION
**Pull request by euhmeuh**

### CODE REVIEW

File: packages/manager/apps/public-cloud/src/index.html [hunk 1]
```diff
@@ -38,7 +38,14 @@ <h1 class="mt-3 mb-5" data-ng-if="('preload_manager_ovh' | translate) !== 'prelo
                 </div>
             </div>
             <div data-otrs-popup></div>
-            <ovh-chatbot></ovh-chatbot>
+            <ovh-chatbot
```
Comment (FredericEspiau) [inline packages/manager/apps/public-cloud/src/index.html]: You remove the import but still use it ? Maybe I'm missing something :)

Comment (euhmeuh) [reply packages/manager/apps/public-cloud/src/index.html]: :sparkles: It's magic. :sparkles:

**Comment by jleveugle** [comment]
Hello @euhmeuh,
Your pull request is now closed due to inactivity.
Feel free to re-open it if you have any update.
Anna Bot for OVH UX <3

**Comment by jleveugle** [comment]
Hello @euhmeuh,
Your pull request is now closed due to inactivity.
Feel free to re-open it if you have any update.
Anna Bot for OVH UX <3

**Comment by jleveugle** [comment]
Hello @euhmeuh,
Your pull request is now closed due to inactivity.
Feel free to re-open it if you have any update.
Anna Bot for OVH UX <3

**Pull request closed without merging**